### PR TITLE
Add de.syping.gta5view

### PR DESCRIPTION
**Pull request by Syping**
[gta5view](https://gta5view.syping.de/) is a Open Source Snapmatic picture and Savegame viewer/editor for [GTA V](https://gta.fandom.com/wiki/Grand_Theft_Auto_V).
GitHub Repo is [https://github.com/SyDevTeam/gta5view/](https://github.com/SyDevTeam/gta5view/).

gta5view is licensed with the [GNU GPL 3.0+](https://github.com/SyDevTeam/gta5view/blob/1.8.x/LICENSE) license.

### CODE REVIEW

**Comment by bilelmoussaoui** [comment]
bot, build de.syping.gta5view

File: de.syping.gta5view.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    config-opts:
+      - -DFLATPAK_BUILD=ON
+      - -DQCONF_BUILD=ON
+    sources:
+      - type: git
+        url: https://github.com/SyDevTeam/gta5view.git
+        tag: 1.8.0
+        commit: 547922329afbd8b35d63d81ea5772563a1c24fe2      
+      - type: file
+        path: de.syping.gta5view.appdata.xml
```
Comment (LeandroStanger) [inline de.syping.gta5view.yaml:24]: New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

**Review by Syping — COMMENTED** [review]
These changes right?

File: de.syping.gta5view.metainfo.xml [hunk 2]
```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (barthalion) [inline de.syping.gta5view.metainfo.xml:1]: This belongs upstream, so you can also updated `<releases>` section as part of your release procedure. The file is not specific to Flathub and other "stores" (Linux distros or snap) can also use it.

File: de.syping.gta5view.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+app-id: de.syping.gta5view
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: gta5view
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=x11
```
Comment (barthalion) [inline de.syping.gta5view.yaml:9]:
```suggestion
  - --socket=fallback-x11
```

**Comment by barthalion** [comment]
bot, build de.syping.gta5view

**Comment by Syping** [comment]
Upstreamed metainfo and updated the commit

**Comment by barthalion** [comment]
bot, build de.syping.gta5view

**Pull request closed without merging**